assets/webpack.config.js

```diff
@@ -40,5 +45,6 @@ module.exports = (_env, _options) => ({
   plugins: [
     new MiniCssExtractPlugin({filename: '../css/app.css'}),
     new CopyWebpackPlugin({patterns: [{from: 'static/', to: '../'}]})
-  ]
+  ],
```

Comment:
```js
].concat(devMode ? [new HardSourceWebpackPlugin()] : [])
```

But I personally had many issues with `HardSourceWebpackPlugin` corrupting it’s cache multiple times a day…